frontend/taipy-gui/src/components/Taipy/Image.tsx

```diff
@@ -58,12 +58,15 @@ const Image = (props: ImageProps) => {
         return [undefined, undefined, false];
     }, [content]);
 
-    const style = useMemo(() => ({
-        width: width,
-        height: height,
-        display: inlineSvg ? "inline-flex" : undefined,
-        verticalAlign: inlineSvg ? "middle" : undefined
-    }), [width, height, inlineSvg]);
+    const style = useMemo(
```

Comment:
Still formatting differences!

Reply:
Might need to add a prettierrc file so that we could have a common formatting style